ui/src/components/dataset/upload/UploadDatasetStepper.vue

```diff
@@ -1,6 +1,5 @@
 <template>
```

Comment:
Changes:
- Move file-selection to first step
- Provide fields to select uploaded dataset's type, source Raw Data, associated Project and source instrument in a single step
- Provide ability to choose the desired dataset name via an input field